lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -66,11 +157,11 @@ import kotlin.collections.*
  * @param compileOptions Various options that effect how the source code is compiled.
  */
 internal class EvaluatingCompiler(
-    private val valueFactory: ExprValueFactory,
-    private val functions: Map<String, ExprFunction>,
-    private val customTypedOpParameters: Map<String, TypedOpParameter>,
-    private val procedures: Map<String, StoredProcedure>,
-    private val compileOptions: CompileOptions = CompileOptions.standard()
+        private val valueFactory: ExprValueFactory,
```

Comment:
Hmm most of the lines changed in this file (other than the imports) add some extra whitespace. I'm concerned this will make it harder to view the git history/blame for these adjusted lines. Could the extra whitespace be reverted from this PR?

Reply:
Will fix.

Reply:
Looks like the reason for these was that my IDE's continuation indent size has been 8 spaces. I fixed that.

Reply:
Ok, it's good we found the cause!